src/bonds.cpp

```diff
@@ -491,6 +638,24 @@ TEST_CASE("[Faunus] BondData") {
             bond.setEnergyFunction(p_60deg_4a);
             CHECK_EQ(bond.energyFunc(distance), Approx(100.0 / 2 * std::pow(cos(60.0_deg) - cos(45.0_deg), 2)));
         }
+        SUBCASE("GromosTorsion Force") {
+            GromosTorsion bond(100.0, cos(45.0_deg), {0, 1, 2});
```

Comment:
`std::cos()`

Reply:
Fixed particular and other instances with f1feccc.